### ISSUE TITLE
[Suggestion] Add a flag to `distrobox-host-exec` to skip the flatpak-spawn confirmation message.

### ISSUE THREAD

**Issue by emmetoneillpdx** [89luca89/distrobox#362]
**Is your feature request related to a problem? Please describe.**

I have a setup where a script inside of my distrobox container is using `distrobox-host-exec` to open a browser on the host (with `gio open [url]`). I understand that it's better to use `flatpak-spawn` to achieve what I'm going for here, but it isn't available in the slightly outdated version of Ubuntu LTS that my container is based on. This container is designed to be shared with other people however, and I feel that the forced flatpak-spawn confirmation message is a bit annoying in that context.

**Describe the solution you'd like**

I'd like to be able to pass a CLI flag to `distrobox-host-exec` (for example, "-y" or whatever) to have it automatically skip past the confirmation message. 

**Describe alternatives you've considered**

For now I'll just have my users deal with the message, but it'd be nicer if they didn't have to.

**Additional context**

We've been putting together a distrobox-based development container for the FOSS painting and animation application, Krita. :smile: 

https://invent.kde.org/eoinoneill/krita-devbox

PS: Distrobox is excellent! :+1:

**Comment by codexMechanicus** [comment]
One way to get around the prompt is to pipline a yes.
Like this:
 `yes | distrobox-host-exec`

**Comment by codexMechanicus** [comment]
Update I've implemented the changes discussed here by [emmetoneillpdx](https://github.com/emmetoneillpdx) in my fork and submitted a merge request.

I've tried to keep the code high quality, and confirming to distrobox's conventions with comments and submitted a pull request with the changes. 
I added a --no-confirm/-nc flag. it doesn't force using host-spawn it just skips the prompt and tries to install host-spawn if flatpak and host-spawn are not installed.

I updated the feedback and added information about the --no-confirm flag to the --help flags feedback.
